lms/djangoapps/branding/api.py

```diff
@@ -218,12 +221,29 @@ def _build_support_form_url(full_path=False):
     return contact_us_page
 
 
+def _build_help_center_url(language=settings.LANGUAGE_CODE):
+    """
+    Return the help-center URL based on the
+    language selected on the homepage.
+    :param language: selected language
+    :return: help-center URL
+    """
+    support_url = settings.SUPPORT_SITE_LINK
+    enabled_languages = {
+        'en': 'hc/en-us',
+        'es': 'hc/es-419'
```

Comment:
Also, I believe the language code for Spanish will be `es_419` instead `es` only.

Reply:
changed `es` to `es-419`.